.github/workflows/upload-demo-archive-and-trigger-examples-repository-update.yml

```diff
@@ -56,6 +56,6 @@ jobs:
           client-payload: '{
               "build_demo_repo": "${{ github.repository }}",
               "build_demo_workflow_id": "upload-demo-archive.yml",
-              "artifact_name": "demo-${{github.sha}}",
+              "artifact_name": "demo-${{ github.sha }}",
```

Comment:
the issue is not in the spaces between the braces and the value, it is probably in the variable interpolation and the place of the double quotes

Reply:
On the playground repositry, I used ` "demo_sha": "${{ github.sha }}"`, and it works.
When I tested the Github action, I read if we don't have the spaces between the braces and the value, the value is not calculated. And I already had this error.

Reply:
- https://github.com/process-analytics/github-actions-playground/blob/master/.github/workflows/build_demo.yml
- https://github.com/process-analytics/github-actions-playground/blob/master/.github/workflows/download_demo.yml

Reply:
the difference with the playground is here we want to do string concatenation with a constant `demo-` and context expansion, IMHO this is what needs to be retested in the playground. Maybe the spaces are important as well, but I have this example that works without spaces: https://github.com/tbouffard/playground-release-drafter-and-gh-pages/blob/0d4dae4d5bc7e88319e616b6e3178c2d8c289b88/.github/workflows/gh-pages-publishing.yml#L44

Reply:
so the issue was also related to bad param:
"build_demo_workflow_id": "upload-demo-archive-and-trigger-examples-repository-update.yml",